src/Core/System/Snippet/SnippetFixer.php

```diff
@@ -15,7 +18,19 @@ public function __construct(private readonly SnippetFileHandler $snippetFileHand
     {
     }
 
+    /**
+     * @deprecated tag:v6.8.0 reason:new-optional-parameter - Will get a second parameter `$invalidPluralization`
+     */
     public function fix(MissingSnippetCollection $missingSnippetCollection): void
```

Comment:
please already add the new parameter with a comment 

```suggestion
    public function fix(MissingSnippetCollection $missingSnippetCollection /*, array $invalidPluralization*/): void
```